cmake/zephyr_module.cmake

```diff
@@ -48,6 +48,8 @@ if(WEST OR ZEPHYR_MODULES)
       message(FATAL_ERROR "${zephyr_module_error_text}")
   endif()
 
+  set(MODULE_ROOT ${ZEPHYR_BASE}/modules)
```

Comment:
Should the comment in the top of this file be updated to include a description of the `MODULE_ROOD` functionality?

Reply:
It should.

Reply:
done.
please make additional comments in #30904